instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/utils/HttpUrlUtils.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.api.tracer.utils;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.attributes.SemanticAttributes;
+import java.net.URI;
+
+public class HttpUrlUtils {
+
+  public static void setHttpUrlAttribute(Span span, URI url) {
```

Comment:
I don't think we need `Attribute` in the names, it's sort of an implementation detail. We're mostly concerned with setting the URL into the Span

Reply:
how about setNetPeer for NetPeerUtils to be consistent with setHttpUrl in HttpUrlUtils?

Reply:
How about?
* `HttpUrlUtils.setHttpUrl()`
* `NetPeerUtils.setNetPeerInfo()`

(the `info` because it sets more than one attribute)